_source/_posts/2018-06-28-tutorial-build-a-basic-crud-app-with-node.md

```diff
@@ -1164,30 +1152,30 @@ router.get("/:slug", (req, res, next) => {
 
 ## Test Your New CRUD App!
 
-By this point, you've built a fully functional Node.js website using Express.js and Okta. To test it out, run the following command to start up your web server then visit `http://localhost:3000` in the browser.
+By this point, you've built a fully functional Node.js website using Express.js and Okta. Run the following command to start your web server, then visit `http://localhost:3000` in the browser to test it out.
 
 ```bash
 npm start
 ```
 
-If you were able to copy the code properly, you should be able to log in, create posts, edit posts, and delete posts.
+If you could copy the code correctly, you should be able to log in, create posts, edit posts, and delete posts.
```

Comment:
i think this sentence might be to be re-evaluated. should it be 
> If you copied the code correctly, ...

or maybe something else less judgey? :)

Reply:
Thumbs up on Alisa's edit here. You could go a little further to make it more concise: "If you copied the code correctly, you should be able to log in to create, edit, and delete posts. "